test/data/marriage-abroad-questions-and-responses.yml

```diff
@@ -29,6 +29,7 @@
 - cyprus
 - czech-republic
 - denmark
+- dominican-republic
```

Comment:
While it's good to have small commits, that idea can be taken too far. I prefer to see an atomic & coherent set of changes per commit. I think this commit is probably too small and should probably be combined with others to make a coherent set of changes.

I think it would also help if the commit note explained **why** you are making this change.